src/_includes/style.css

```
@@ -1,13 +1,40 @@
+html {
+  --primary-text-color: #212121;
+  --secondary-text-color: #727272;
+  --accent-text-color: #212121;
+  --primary-background-color: #fafafa;
+  --secondary-background-color: #ffffff;
+  --divider-color: rgba(0, 0, 0, 0.12);
+  --primary-color: #03a9f4;
+  --accent-color: #ffc900;
+}
+
+@media (prefers-color-scheme: dark) {
+  html {
+    --primary-text-color: #e1e1e1;
+    --secondary-text-color: #9b9b9b;
+    --primary-background-color: #111111;
+    --secondary-background-color: #202020;
```

Let's try to go with this

```suggestion
    --primary-background-color: #202020;
    --secondary-background-color: #303030;
```